packages/office-addin-manifest/test/test.ts

```diff
@@ -489,11 +489,11 @@ describe("Unit Tests", function() {
         assert.strictEqual(info.defaultLocale, "en-us");
         assert.strictEqual(info.description, "A template to get started.");
         assert.strictEqual(info.displayName, "Contoso Task Pane Add-in");
-        assert.strictEqual(info.highResolutionIconUrl, "https://localhost:3000/assets/icon-128.png");
```

Comment:
What made this necessary?

Reply:
I updated the test manifest to using local references to icons to validate their inclusion in the zip file for json manifest add-ins.  This then validates those paths rather than the localhost paths.  I also added the extra parameter to distinguish between this and the other entry with the same value.